proto/viam/app/v1/app.proto

```diff
@@ -708,18 +723,27 @@ message RemoveRoleRequest {
 
 message RemoveRoleResponse {}
 
+message Authorizations {
+  string identity_id = 1;
+  repeated Authorization authorizations = 2;
+}
+
 message ListAuthorizationsRequest {
   string resource_type = 1;
   string resource_id = 2;
   string organization_id = 3;
 }
 
-message Authorizations {
-  string identity_id = 1;
-  repeated Authorization authorizations = 2;
+message ListAuthorizationsResponse {
+  repeated Authorizations authorizations = 1;
 }
 
-message ListAuthorizationsResponse {
+message ListAuthorizationsForUserRequest {
+  string user_id = 1;
+  string organization_id = 3;
```

Comment:
actually, should we omit the query `organization_id` and use the current org on the state? or should we make this explicit

Reply:
let's make it explicit - according to Michael, currentOrg isn't always populated correctly